npm-shrinkwrap.json

```diff
@@ -8223,8 +8223,9 @@
       "resolved": "https://registry.npmjs.org/stylis-rule-sheet/-/stylis-rule-sheet-0.0.10.tgz"
     },
     "sudo-prompt": {
-      "version": "8.0.0",
-      "resolved": "https://registry.npmjs.org/sudo-prompt/-/sudo-prompt-8.0.0.tgz"
+      "version": "8.2.0",
+      "from": "zvin/sudo-prompt#use-localhost",
+      "resolved": "git://github.com/zvin/sudo-prompt.git#1563b42dce5c48617af1b250a9343d63a26a5fa1"
```

Comment:
Does the commit SHA need updating, now that `sudo-prompt#use-localhost` has been updated?

Reply:
updated